torchci/rockset/commons/__sql/weekly_force_merge_stats.sql

```diff
@@ -96,54 +103,43 @@ WITH
             ) AS force_merge_with_failures,
             CAST(time as DATE) as date
         FROM
-            force_merges_with_failed_checks
+            merges_identifying_force_merges
         ORDER BY
             date DESC
     ),
-    stats_per_day as (
+    bucketed_counts as (
         select
-            count(*) as total,
-            sum(force_merge) as total_force_merge_cnt,
-            sum(force_merge_with_failures) as with_failures_cnt,
-            sum(force_merge) - sum(force_merge_with_failures) as impatience_cnt,
-            date,
+            IF(
+                :one_bucket,
+                'Overall',
+                FORMAT_TIMESTAMP('%Y-%m-%d', DATE_TRUNC(:granularity, date))
+            ) AS granularity_bucket,
+            SUM(force_merge_with_failures) with_failures_cnt,
```

Comment:
There is a missing "as" keyword here, which makes the "From Failures" value on the KPIs page https://torchci-git-zainr-force-metrics-fbopensource.vercel.app/kpis wrong I think (~25%)